far/vmenu.cpp

```diff
@@ -140,15 +140,45 @@ void VMenu::ResetCursor()
 	GetCursorType(PrevCursorVisible,PrevCursorSize);
 }
 
+template <typename coll, typename pred>
+int FindNearestSelectableItem(const coll& Coll, const int Pos, pred&& Pred, const bool GoBackward, const bool DoWrap)
```

Comment:
Yeah, that's true. What do you think of `FindNearestAcceptableItem`? Another option I can think of is `FindNearestMatchingItem`, but "matching" here feels misleading.

The function is generic mostly for the sake of testing, otherwise we could hardcode `Itms` and `ItemFlagsAllowFocus`. I think it makes sense to keep it generic.

Reply:
Just FindNearest maybe?